teeio-validator/library/cxl_ide_test_lib/cxl_ide_common.c

```diff
@@ -87,36 +87,36 @@ ide_test_case_name_t m_cxl_ide_test_case_names[] = {
 };
 
 ide_test_case_funcs_t m_cxl_ide_query_cases[MAX_CXL_QUERY_CASE_ID] = {
-  {cxl_ide_test_query_1_setup, cxl_ide_test_query_1_run, cxl_ide_test_query_1_teardown, false},
-  {cxl_ide_test_query_2_setup, cxl_ide_test_query_2_run, cxl_ide_test_query_2_teardown, false},
+  {cxl_ide_test_query_1_setup, cxl_ide_test_query_1_run, cxl_ide_test_query_1_teardown, false, 6},
```

Comment:
it is too hard to maintain this assertion_cnt here.

Reply:
Will update in next version.